packages/teleport/src/Desktops/Desktops.tsx

```diff
@@ -64,6 +71,26 @@ export function Desktops(props: State) {
           />
         </>
       )}
+      {isEmpty && (
+        <Empty
+          clusterId={clusterId}
+          canCreate={canCreate && !isLeafCluster}
+          onClick={() => window.open(DOC_URL)}
```

Comment:
i think it would be confusing (or maybe see it as a bug?) if the user clicks on `Add Desktops` only to lead to the same link as `View Documentation`? your call, but we have to make sure we set `noreferrer` on all links

Reply:
Yeah, I was following the pattern set by the k8s page, but you're right that this could be interpreted as a bug. I'll remove the "Add x" buttons from both pages for now and leave the "View Documentation" button.